libs/langgraph/langgraph/prebuilt/chat_agent_executor.py

```diff
@@ -530,6 +561,7 @@ def should_continue(state: AgentState) -> Literal["tools", "__end__"]:
 
     # Define the function that calls the model
     def call_model(state: AgentState, config: RunnableConfig) -> AgentState:
+        _validate_messages(state["messages"])
```

Comment:
Is it possible for the messages to ever end up being dict representation of open ai format requests? (i.e., not base messages)

Reply:
no, the messages go through `add_messages` reducer